src/pages/Members/Members.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import './styles.scss';
+
+import memberGroupIcon from '@/assets/icons/member-group.svg';
+import MemberButtons from '@/components/TopBar/components/MemberButtons/MemberButtons';
+import { useWorkspace } from '@/hooks/useWorkspace';
+
+const Members = () => {
+  const { members, activeWorkspace } = useWorkspace();
+
+  const renderHeader = () => {
+    return (
+      <div className="members-header">
```

Comment:
This header isn't looking like the other pages, you could re-use this https://github.com/EO-DataHub/eodhp-workspace-ui/blob/main/src/pages/Stores/Stores.tsx#L16